Bottom: dd9f1256dd06403294e07aed0a73df0499d092e3
Top:    bb5903f98b0d885b3500e797d94b1d11985625c2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-26 01:28:56 +0000

Refresh of clear-pins-and-vias-last

---

diff --git a/src/polygon.c b/src/polygon.c
index 08ce69d..65879c8 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -872,7 +872,7 @@ subtract_accumulated (struct cpInfo *info, PolygonTypePtr polygon)
 {
   if (info->accumulate == NULL)
     return;
-  Subtract (info->accumulate, polygon, True);
+  Subtract (info->accumulate, polygon, true);
   info->accumulate = NULL;
   info->batch_size = 0;
 }
